packages/terra-clinical-onset-picker/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "terra-clinical-onset-picker",
+  "version": "0.1.0",
+  "description": "A complete form for selecting an onset date. Dates can be selected with a precision of day, month, or year.",
```

Comment:
You excluded the age precision from this description.

Reply:
It would be good to sync the descriptions here and in the README.

Reply:
I've changed to description to match the README.